src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.diff.sql

```diff
@@ -0,0 +1,42 @@
+﻿/***************************************************************************************/
+-- STORED PROCEDURE
+--     AddExtendedQueryTags
+--
+-- DESCRIPTION
+--    Add a list of extended query tags.
+--
+-- PARAMETERS
+--     @extendedQueryTags
+--         * The extended query tag list
+--     @maxCount
+--         * The max allowed extended query tag count
+--     @ready
+--         * Indicates whether the new query tags have been fully indexed
+/***************************************************************************************/
+ALTER PROCEDURE dbo.AddExtendedQueryTags (
+    @extendedQueryTags dbo.AddExtendedQueryTagsInputTableType_1 READONLY,
+    @maxAllowedCount INT,
+    @ready BIT = 0
```

Comment:
should we use `TagStatus` (TINYINT) here instead of `@ready ` (BIT) to make it more flexible? since it's not easy to change SQL.
Also avoid the conversion from BIT to TINYINT which adds more confusion to the status mapping.

Reply:
Well, this is supposed to be a boolean flag between ready and not ready. It's just a nice coincidence that the bits map nicely to the appropriate tag status. I'd be worried about using tinyint to give callers even more values to choose from, as we presumably don't want to give callers the ability to add just any ol' status into the table